Source/Parallelization/WarpXComm.cpp

```diff
@@ -293,6 +312,18 @@ WarpX::UpdateAuxilaryDataStagToNodal ()
 void
 WarpX::UpdateAuxilaryDataSameType ()
 {
+    // Update aux field, including guard cells, up to ng_FieldGather
+    const amrex::IntVect& ng_src = guard_cells.ng_FieldGather;
+
+    // Level 0: Copy from fine to aux
+    // TODO Check if aux and fp are aliases of each other. If so, we can skip the copy.
```

Comment:
Not needed anymore after this PR is merged:
https://github.com/AMReX-Codes/amrex/pull/3986